src/layout.rs

```diff
@@ -34,6 +35,12 @@ pub enum Constraint {
     Min(u16),
 }
 
+impl Default for Constraint {
+    fn default() -> Constraint {
+        Constraint::Percentage(100)
+    }
+}
```

Comment:
I wonder if Constraint makes sense to have a Default. Where would you see this being used?

Reply:
I just saw this is more widely used, but it's OK to remove it if not so helpful.